libraries/chain/protocol/transaction.cpp

```diff
@@ -103,6 +103,7 @@ void transaction::get_required_authorities( flat_set<account_id_type>& active, f
 
 
 
+const fc::flat_set<public_key_type> empty_keyset;
```

Comment:
I thought it's `boost::flat_set`. It's actually a fc wrapper? Perhaps just remove `fc::` here, since everywhere else doesn't have it.